CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 - Adding support for Helix QAC 2021.1[\#755](https://github.com/eclipse-iceoryx/iceoryx/issues/755) thanks to @toniglandy1
 - Axivion analysis on CI[\#409](https://github.com/eclipse-iceoryx/iceoryx/issues/409)
 - Cpptoml can be provided by an external source[\#951](https://github.com/eclipse-iceoryx/iceoryx/issues/)
+- Extend cxx::optional constructor for in place construction so that copy/move for values inside the optional even could be deleted[\#967](https://github.com/eclipse-iceoryx/iceoryx/issues/967)
```

Comment:
The sentence was not quite clear to me at first. I think it should more accurately state that copy or move constructor for the types wrapped by optional are now allowed to be deleted ("values" is not accurate and was confusing me at first).

I.e. this drops a previously existing restriction were this was not possible.